vg/vgimg/vgimg.go

```diff
@@ -111,8 +117,9 @@ func NewWith(o ...option) *Canvas {
 // used when initializing a canvas are compatible with
 // each other.
 const (
-	setsDPI = 1 << iota
-	setsSize
+	setsDPI        uint32 = 1
```

Comment:
`setsDPI uint32 = 1 << iota` and remove the assignments below.

Reply:
But then, would `setsBackground` == 3? I believe I tested and confirmed that if it was 3 rather than 4 then things wouldn't work correctly, but I may be wrong.

Reply:
https://play.golang.org/p/2qcq_z_KJ0S